File: HW07_CS/Rectangle.cs
namespace HW07
{
    class Rectangle{
        public double Width { get; set; } = 1.0;
        public double Height { get; set; } = 1.0;

        public double Area => Width * Height;

        public void RectangleInfo(){
            System.Console.WriteLine($"Width: {Width:N2}");
            System.Console.WriteLine($"Height: {Height:N2}");
            System.Console.WriteLine($"Area: {Area:N2}");
        }
    }
}

File: HW07_PY/Rectangle.py
class Rectangle:
    def __init__(self, width=1.0, height=1.0):
        self.__width = width
        self.__height = height

    @property
    def width(self):
        return self.__width

    @width.setter
    def width(self, value):
        self.__width = value

    @property
    def height(self):
        return self.__height

    @height.setter
    def height(self, value):
        self.__height = value

    def area(self):
        return self.__width * self.__height

    def RectangleInfo(self):
        print(f"Width: {self.width}")
        print(f"Height: {self.height}")
        print(f"Area: {self.area():.2f}")